show/main.py

```diff
@@ -1803,20 +1803,74 @@ def mmu():
 # 'reboot-cause' command ("show reboot-cause")
 #
 @cli.command('reboot-cause')
-def reboot_cause():
-    """Show cause of most recent reboot"""
-    PREVIOUS_REBOOT_CAUSE_FILE = "/host/reboot-cause/previous-reboot-cause.txt"
-
-    # At boot time, PREVIOUS_REBOOT_CAUSE_FILE is generated based on
-    # the contents of the 'reboot cause' file as it was left when the device
-    # went down for reboot. This file should always be created at boot,
-    # but check first just in case it's not present.
-    if not os.path.isfile(PREVIOUS_REBOOT_CAUSE_FILE):
-        click.echo("Unable to determine cause of previous reboot\n")
+@click.argument('history', required=False, type=click.Choice(["history"]))
+def reboot_cause(history):
```

Comment:
It seems like "reboot_cause" should now become a group and "history" should be a subcommand of that group. The "reboot_cause" group should be defined with `invoke-without-command=True` and the default implementation should be to read and display the previous reboot cause.